modules/monitoring-about-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-creating-alerting-rules-for-user-defined-projects_{context}"]
+= About creating alerting rules for user-defined projects
+
+If you create alerting rules for a user-defined project, consider the following key behaviors and important limitations when you define the new rules:
+
+* A user-defined alerting rule can include metrics exposed by its own project in addition to the default metrics from core platform monitoring. 
+You cannot include metrics from another user-defined project.
++
+For example, an alerting rule for the `ns1` user-defined project can use metrics exposed by the `ns1` project in addition to core platform metrics, such as CPU and memory metrics.
+However, the rule cannot include metrics from a different `ns2` user-defined project.
+
+* To reduce latency and to minimize the load on core platform monitoring components, you can add the `openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus` label to a rule to force only the Prometheus instance deployed in the `openshift-user-workload-monitoring` project to evaluate the alerting rule instead of the Thanos Ruler instance.
++
+[IMPORTANT]
+====
+If an alerting rule has this label, your alerting rule can use only those metrics exposed by your user-defined project.
+Alerting rules you create based on default platform metrics might not fire.
```

Comment:
```suggestion
Alerting rules you create based on default platform metrics might not start.
```

As per IBMSG, see "fire": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f

Reply:
Changed to `trigger`, although the IBM Style entry is for `fire up`, not `fire`. For alerts, I think that `trigger` is more accurate than `start` as a substitute here.